src/Layout/ComputationMenu/ComputationMenu.tsx

```diff
@@ -278,16 +274,14 @@ export class ComputationMenu extends React.Component<ComputationMenuProps, {}> {
      */
     computeManually = () => {
         const year = parseInt(this.props.settingsPayload.year);
-        const election = this.props.electionType.elections.find((e) => e.year === year);
         const votes = this.props.votes.filter((vote) => vote.electionYear === year);
         const distributionYear = this.props.use2021Distribution && year >= 2005 ? 2021 : year;
         const metrics = this.props.metrics.filter((metric) => metric.electionYear === distributionYear);
         const parameters =
             this.props.parameters.find((parameter) => parameter.electionYear === year) || unloadedParameters;
-        if (election !== undefined && election !== null) {
+        if (parameters !== undefined) {
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5976811)